source/application/DbgHelp/DbgHelpTypes.h

```diff
@@ -10,7 +10,14 @@
 #include <string>
 #include <variant>
 
+/*
+ * Make the pointless warning shut up. This can be removed when the bug below is fixed.

```

Comment:
Can we reword this to something like "This triggers the warning erroneously due to a bug: -link-"?